pytest_fixtures/component/subscription.py

```diff
@@ -12,12 +12,16 @@ def subscribe_satellite(clean_rhsm, default_sat):
     """subscribe satellite to cdn"""
     from robottelo.config import settings
 
+    if default_sat.os_version.major < 8:
+        release_version = f'{default_sat.os_version.major}Server'
+    else:
+        release_version = f'{default_sat.os_version.major}'
```

Comment:
Optional: Changing this to one-liner would be nice, like,
```suggestion
    major_version = default_sat.os_version.major
    release_version = f'{major_version}Server' if major_version < 8 else f'{major_version}'
```